public/admin/cmsEncryptionWidget.js

```diff
@@ -0,0 +1,143 @@
+const EMAIL_CMS_FIELD = 'email'
+const FULL_NAME_CMS_FIELD = 'full-name'
```

Comment:
Use Route to check field names

Reply:
Use the GET route to return the name of the CMS fields using CMS utils functions.